index.html

```diff
@@ -9222,6 +9222,240 @@ <h3 id="example-2">
+                </li>
+                <li>Else, multiply <var>compressor gain</var> by <var>enveloppe
+                rate</var>
+                </li>
+                <li>Compute <var>final gain</var> to be <var>compressor
+                gain</var> multiplied by the return value of <a>computing the
+                makeup gain</a>.
+                </li>
+                <li>Compute <var>metering gain</var> to be <var>final
+                gain</var>, <a href="#linear-to-decibel">converted to
+                decibel</a>.
+                </li>
+              </ol>
+            </li>
+          </ol>
+          <p>
+            The makeup gain is a fixed gain stage that only depends on ratio,
+            knee and threshold parameter of the compressor, and not on the
+            input signal. The intent here is to increase the output level of
+            the compressor so it is comparable to the input level.
+          </p>
+          <p>
+            Computing the makeup gain means executing the following steps:
+          </p>
+          <ol>
+            <li>Let <var>full range gain</var> be the value returned by putting
+            1.0 through the <a>compression curve</a>.
+            </li>
+            <li>Let <var>full range makeup gain</var> the inverse of <var>full
+            range gain</var>.
+            </li>
+            <li>Return the result of computing <var>full range makeup
+            gain</var><sup>0.6</sup>.
+            </li>
+          </ol>
+          <p>
+            Computing the <dfn>enveloppe rate</dfn> is done by putting the
+            ratio of the <var>compressor gain</var> and the <var>detector
+            average</var> through a curve. User-agents are allowed to choose
+            the shape of the attack and release curve. However, they MUST
+            respect the following constraints:
+          </p>
+          <ul>
+            <li>The input to the curve MUST be the ratio of the <var>compressor
+            gain</var> and the <var>detector average</var>.
+              <div class="note">
+                When attacking, this number inferior or equal to 1, when
+                releasing, this number is strictly superior to 1.
+              </div>
+            </li>
+            <li>The attack curve MUST be a continuous, monotonically increasing
+            function in the range \([0, 1]\).
+            </li>
+            <li>The release curve MUST be a continuous, monotonically
+            decreasing function that is always superior to 1.
+            </li>
+          </ul>
+          <div class="note">
+            It is allowed, for example, to have a compressor that performs an
+            <em>adaptative release</em>, that is, releasing faster the harder
+            the compression, or to have curves for attack and release that are
+            not of the same shape.
+          </div>
+          <p>
+            Putting a the absolute value of a sample through a <dfn>compression
```

Comment:
You use the phrase "putting foo through a curve" in many places above.  Maybe add a `<dfn>` for this and reference it?